src/components/nav/allRoutesSwitch.tsx

```diff
@@ -0,0 +1,10 @@
+import { Switch } from "../../utils/Switch";
+import { Routes } from "./Routes";
+import { Router } from "./Router";
+
+export const allRoutesSwitch = 
```

Comment:
this thing just builds up the same Switch that was originally in the `Site`, but it does it by `reducing` a bunch of separately defined route cases into a large one, returning on each reduction a more and more configured switch with more and more possibly matching cases.